test/hotspot/jtreg/serviceability/dcmd/vm/SystemMapTest.java

```diff
@@ -27,6 +27,18 @@
 import jdk.test.lib.dcmd.JMXExecutor;
 import jdk.test.lib.process.OutputAnalyzer;
 
+/*
+ * @test
+ * @summary Test of diagnostic command System.map using ZGC
+ * @library /test/lib
+ * @requires (os.family == "linux" | os.family == "windows" | os.family == "mac")
+ * @modules java.base/jdk.internal.misc
+ *          java.compiler
+ *          java.management
+ *          jdk.internal.jvmstat/sun.jvmstat.monitor
+ * @run testng/othervm -XX:+UsePerfData -XX:+UseZGC SystemMapTest
```

Comment:
No need to add separate testcase for ZGC. It is expected that  hotspot tests are usually executed to ensure that all functionality works with any GC.
If you want to add it to ensure that  regression testcase is always executed even no ZGC is set then please 
1) add
`* @requires vm.gc.Z`
So testcase is not executed if non-ZGC GC (Parallel/Serial etc) is set explicitly.
2) Add 
` * @bug 8346717`
3) Add id like
`* @test id=zgc`
to make testcase name clearer.